packages/react-ui-ag/src/Filters/__tests__/PriceFilterCard-test.js

```diff
@@ -35,4 +37,24 @@ describe('ag/Filters/PriceFilterCard', () => {
     expect(wrapper.find('[data-tid="foo"]')).toHaveLength(1)
     expect(wrapper.find('[data-tid="price-filter-card"]')).toHaveLength(0)
   })
+
+  it('calls onChange when <RangeSlider/> onChangeComplete event occurs', () => {
+    const onChangeFn = jest.fn()
+    const wrapper = mount(
```

Comment:
Do we need to `mount` here?

Reply:
He's searching for and calling a function a couple levels down, so in this case I think yes.